### PR TITLE
[Zaraz] Added new triggers

### PR DESCRIPTION
**Pull request by marciocloudflare**
Addresses PCX-4470

### CODE REVIEW

File: content/zaraz/reference/triggers.md [hunk 1]
```diff
@@ -127,6 +127,44 @@ Refer to [**Create a trigger**](/zaraz/get-started/create-trigger/) to learn how
...
 
+### Element Visibility
+
+Triggers an action when a CSS selector becomes visible in the screen.
+
+{{<table-wrap>}}
+
+| Rule type            | CSS Selector |
+| -------------------- | ------------ |
+| _Element Visibility_ | `#my-class` |
```
Comment (jonnyparris) [inline content/zaraz/reference/triggers.md]:
```suggestion
| _Element Visibility_ | `#my-id` |
```
class would be prefixed with `.` not a `#`

**Review by jonnyparris — CHANGES_REQUESTED** [review]
minor semantic change

File: content/zaraz/reference/triggers.md [hunk 2]
```diff
@@ -127,6 +127,44 @@ Refer to [**Create a trigger**](/zaraz/get-started/create-trigger/) to learn how
 
 {{</table-wrap>}}
 
+### Element Visibility
```
Comment (pedrosousa) [inline content/zaraz/reference/triggers.md:130]:
```suggestion
### Element visibility
```

File: content/zaraz/reference/triggers.md [hunk 3]
```diff
@@ -127,6 +127,44 @@ Refer to [**Create a trigger**](/zaraz/get-started/create-trigger/) to learn how
...
+
+Triggers an action when a CSS selector becomes visible in the screen.
+
+{{<table-wrap>}}
+
+| Rule type            | CSS Selector |
```
Comment (pedrosousa) [inline content/zaraz/reference/triggers.md:136]:
```suggestion
| Rule type            | CSS selector |
```
If this is a UI label, feel free to ignore the suggestion. :) If you change the text, there are other occurrences in this file, though.

**Review by pedrosousa — APPROVED** [review]

File: content/zaraz/reference/triggers.md (hunk 3 above)
Comment (marciocloudflare) [reply content/zaraz/reference/triggers.md:136]: Yup, those are UI elements :)

**Pull request merged**